src/helpers/utils.ts

```diff
@@ -103,23 +103,54 @@ export function runCommand(cmd: string, requiredRepoType = RepoType.Undefined) {
   } else if (getRepoType() !== requiredRepoType) {
     vscode.window.showErrorMessage(`Command works only with ${requiredRepoType} projects!`);
   } else {
-    const project = resolve(workspace.uri.fsPath, 'package.json');
-
-    try {
-      const { scripts = {} } = __non_webpack_require__(project) || {};
-      const candidates = Object.keys(scripts).filter((m) => scripts[m].trim().startsWith(cmd));
-      const shellCommand =
-        candidates.length === 0 ? cmd : candidates.length === 1 ? scripts[candidates.pop() ?? ''] : undefined;
-
-      if (shellCommand !== undefined) {
-        execCommand(shellCommand);
+    const piralOrPilet = cmd.split(' ')[1];
+    const nodeModulesAreAvailable = existsSync(`${workspace.uri.fsPath}/node_modules`);
```

Comment:
Does not matter, too - only the second check is relevant.

Either we have it all installed or not.

In case the `node_modules/.bin/piral` is not found we can immediately say that dependencies are not installed and do it again.